azdev/operations/setup.py

```diff
@@ -258,14 +264,114 @@ def add_ext_repo(path):
         raise CLIError('Installation aborted.')
 
 
-def setup(cli_path=None, ext_repo_path=None, ext=None, deps=None):
+def _validate_input(cli_path=None, ext_repo_path=None, set_env=None, copy=None, use_global=None):
+    if copy and use_global:
+        raise CLIError("Copy and use global are mutally exlcusive.")
+    if not cli_path or not ext_repo_path:
+        if cli_path == "pypi" and any([use_global, copy, set_env]):
```

Comment:
Does `pypi` an enum value?

Reply:
I don't think it is. In the legacy code it was hardcoded this way. We can make it an enum though if that is the suggestions.

Reply:
Is the value `pypi` input from user?

Reply:
Yes, that is correct. This is the way it has always been.

Reply:
Is there some places to give user a hint that they could input this value? I didn't see it.